### PR TITLE
DEV-8517 homepage getting started content updates

### PR DESCRIPTION
**Pull request by afnazu**
**High level description:**

Updates text and image content and more fine tuning adjustments for the Homepage Getting Started section

**JIRA Ticket:**
[DEV-8517](https://federal-spending-transparency.atlassian.net/browse/DEV-8517)

**Mockup:**
See ticket link

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [x] Design review complete `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Comment by anjenkin** [comment]
I noticed a couple things to consider once we pick this ticket back up - 

image
image

File: src/_scss/pages/homepage/features/_carousel.scss [hunk 1]
```diff
@@ -19,6 +19,14 @@
             @include media($tablet-screen) {
                 display: block;
             }
+            font-size: 28px;
+
+            .fa-chevron-right {
+                color: #555;
```
Comment (anjenkin) [inline src/_scss/pages/homepage/features/_carousel.scss:25]: color can come from _variables.scss

File: src/_scss/pages/homepageUpdate/homepageUpdate.scss [hunk 2]
```diff
@@ -6,8 +6,49 @@
...
+        bottom: -50rem;
+        width: 125vw;
+        margin-left: calc(50% - 50vw);
+
+        background-color: $color-gray-warm-light;
+        opacity: 0.5;
+
+        @include transform(skewY(5deg));
+
+        @include media($medium-screen) {
```
Comment (anjenkin) [inline src/_scss/pages/homepageUpdate/homepageUpdate.scss:24]: All of the places where we have @include media($medium-screen) I think these should be @include media($medium-screen - 1).  At the breakpoints, mainly 992px the images and text overlap but looks good at < 992px and > 992px
Screen Shot 2022-04-18 at 1 17 48 PM
Screen Shot 2022-04-18 at 1 17 27 PM

**Review by anjenkin — CHANGES_REQUESTED** [review]
Looks good!!  I added a few comments about using rem for font-size which we need for accessibility but I didn't add a comment for every instance.  The same for using color variables from _variables.scss instead of the hex value.

File: src/_scss/pages/homepageUpdate/homepageUpdate.scss (hunk 2 above)
Comment (afnazu) [reply src/_scss/pages/homepageUpdate/homepageUpdate.scss:24]: working on a fix for this breakpoint issue!

File: src/_scss/core/_variables.scss [hunk 3]
```diff
@@ -48,6 +48,7 @@ $color-black:                #000;
 $color-base:                 #555;
 $color-link:                 #0071bc;
 
+$color-gray-darker:          #1b1b1b;
```
Comment (anjenkin) [inline src/_scss/core/_variables.scss:51]: Oh sorry, there's already a #1b1b1b in _variables.scss.  I was suggesting using that variable.

File: src/_scss/pages/homepage/features/_carousel.scss (hunk 1 above)
Comment (anjenkin) [reply src/_scss/pages/homepage/features/_carousel.scss:25]: Also here, suggesting using #555 from _variables.scss (ie. $color-base)

**Review by anjenkin — CHANGES_REQUESTED** [review]
two additional comments inline.  otherwise looks good.

**Comment by afnazu** [comment]
> two additional comments inline. otherwise looks good.

thank you, color variables fixed!

**Review by anjenkin — APPROVED** [review]

**Pull request merged**